File: 2176-count-equal-and-divisible-pairs-in-an-array/2176-count-equal-and-divisible-pairs-in-an-array.py
class Solution:
    def countPairs(self, nums: List[int], k: int) -> int:
        count = 0
        for i in range(len(nums)):
            for j in range(i+1,len(nums)):
                if i*j % k == 0 and nums[i] == nums[j]:
                    count +=1
        return count

File: 2176-count-equal-and-divisible-pairs-in-an-array/2176-count-equal-and-divisible-pairs-in-an-array.java
class Solution {
    public int countPairs(int[] nums, int k) {
        int total = 0;
        for (int j = 1; j < nums.length;j++){
            for (int i = 0; i < j; i++){
                if (nums[i] == nums[j] && (i*j)% k == 0){
                    total+=1;
                }
            }
        }
        return total;   
    }
}